pkgs/os-specific/darwin/apple-sdk-11.0/default.nix

```diff
@@ -56,28 +56,51 @@ let
     '';
   };
 
+  mkStdenv = stdenv:
+    let
+      cc = stdenv.cc.override {
+        bintools = stdenv.cc.bintools.override { libc = packages.Libsystem; };
+        libc = packages.Libsystem;
+      };
+    in
+    if stdenv.isAarch64 then stdenv
+    else
+      (overrideCC stdenv cc).override {
+        targetPlatform = stdenv.targetPlatform // {
+          darwinMinVersion = "10.12";
+          darwinSdkVersion = "11.0";
+        };
+      };
+
+  stdenvs = {
+    stdenv = mkStdenv stdenv;
+  } // builtins.listToAttrs (map
+    (v: { name = "clang${v}Stdenv"; value = mkStdenv pkgs."llvmPackages_${v}".stdenv; })
```

Comment:
This cannot be named like this. On Linux all clangXXStdenv are aliases and should fail eval. https://github.com/NixOS/nixpkgs/blob/3be70b091b5af3a3c782bae4af8837267db65bc3/pkgs/top-level/aliases.nix#L216-L226

Reply:
@SuperSandro2000, any idea why `clangXStdenv` is defined in aliases whereas stuff like `clang_X`, `lld_X`, `lldb_X`, and `llvm_X` are defined in `all-packages.nix`?

If there isn't a "good reason" then I think it would make sense to move `clangXStdenv` definitions into `all-packages.nix`.

Reply:
I would start at https://github.com/NixOS/nixpkgs/pull/115288 where the aliases where introduced